examples/devenv-django/devenv.nix

```diff
@@ -0,0 +1,87 @@
+{ pkgs, ... }:
+
+let
+  db_user = "postgres";
+  db_host = "localhost";
+  db_port = "5432";
+  db_name = "db";
+  python_version = "3.10";
+in
+{
+  packages = [ pkgs.git pkgs.postgresql_14 ];
+
+  languages.python = {
+    enable = true;
+    poetry.enable = true;
+  };
+
+  env = {
+    PYTHON_VERSION = python_version;
+    DATABASE_URL = "postgresql://${db_user}@${db_host}:${db_port}/${db_name}";
```

Comment:
Could be a nice addition to the postgres module to automatically set the database url. Though I don't know how standardized this envvar is.

Reply:
A bit of digging shows that this style of database URL is used by ...

- `libpq` https://www.postgresql.org/docs/current/libpq-connect.html#LIBPQ-CONNSTRING ... source https://stackoverflow.com/questions/3582552/what-is-the-format-for-the-postgresql-connection-string-url
- `sqlalchemy` uses pretty much the same style but often includes the database driver like `postgresql+psycopg2://...` https://docs.sqlalchemy.org/en/20/core/engines.html#database-urls
- I use it here via https://github.com/jazzband/dj-database-url which is a dependency of https://github.com/joke2k/django-environ which is used here to load secrets from a `.env` file

I don't know how it applies to other databases but I'd guess it's common